fe/lib/components/JobCategorySuggest/JobCategorySuggestChoices.tsx

```diff
@@ -44,43 +61,61 @@ const JobCategorySuggestChoices = forwardRef<HTMLInputElement, Props>(
         (choice) => choice.jobCategory.id.value === choiceId,
       );
 
-      setSelectedJobCategory(jobCategorySuggest?.jobCategory);
+      setSelectedJobCategory(choiceId);
 
-      if (onSelect && jobCategorySuggest) {
+      if (jobCategorySuggest) {
         onSelect(jobCategorySuggest);
       }
     };
 
     return (
       <Stack space="small">
         <Text weight="strong">Category</Text>
-
         <RadioGroup
           {...restProps}
-          id="job-category-suggest-select"
           name={name}
+          id="job-category-suggest-select"
           onChange={handleChoiceSelect}
-          value={selectedJobCategory?.id.value ?? ''}
+          value={selectedJobCategory ?? ''}
         >
-          {choices.map((choice) => {
-            const { jobCategory, confidence } = choice;
-            const { id } = jobCategory;
-            return (
-              <RadioItem
-                key={id.value}
-                label={getJobCategoryName(jobCategory)}
-                ref={forwardedRef}
-                value={id.value}
-              >
-                {showConfidence && (
-                  <Text tone="secondary" size="small">
-                    <Strong>Confidence score:</Strong>{' '}
-                    {`${(confidence * 100).toFixed(2)}%`}
-                  </Text>
-                )}
-              </RadioItem>
-            );
-          })}
+          <>
+            {suggestions.map((choice) => {
+              const { label, confidence, key } = choice;
+              return (
+                <RadioItem
+                  key={key}
+                  label={label}
+                  ref={forwardedRef}
+                  value={key}
+                >
+                  {showConfidence && (
+                    <Text tone="secondary" size="small">
+                      <Strong>Confidence score:</Strong>{' '}
+                      {`${(confidence * 100).toFixed(2)}%`}
+                    </Text>
+                  )}
+                </RadioItem>
+              );
+            })}
+          </>
+          <RadioItem
+            key={'Other'}
+            label={'Other'}
+            ref={forwardedRef}
+            value={'Other'}
+          >
+            {selectedJobCategory === 'Other' && (
+              <JobCategorySelect
+                client={client}
+                id="job-category-suggest-select-other"
+                onSelect={(jobCategory) => {
+                  onSelect({ jobCategory, confidence: 1 });
```

Comment:
Note that the suggest will only return child categories, but `JobCategorySelect` will return both children and parent. The parent categories aren't suitable for job posting so this would break Ryanair's posting flow.

We can either:

- Pass through the `JobCategoryType` we're getting in this `onSelect` so the consumer (e.g. Ryanair) can ignore `parent` categories.

- Filter out `onSelect` for child categories here completely